.circleci/config.yml

```diff
@@ -390,6 +393,10 @@ commands:
               JRS_OPTIONS="${JRS_OPTIONS} -Djrs.circleci.slack.results=${SLACK_RESULTS}"
             fi
 
+            CI_PARAMETERS=""
+            if [[ "<< parameters.continuous_integration >>" == "true" ]]; then
+              CI_PARAMETERS="${CI_PARAMETERS} $CIRCLE_USERNAME $CIRCLE_BUILD_URL"
```

Comment:
nit: If CI_PARAMETERS is always empty here, then you can directly use `CI_PARAMETERS="$CIRCLE_USERNAME $CIRCLE_BUILD_URL"` to reduce future confusion.